config/sequelize.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: process.env.DB_HOST,
+    dialect: 'mysql'
+  },
+
+  production: {
+    username: process.env.PDB_USERNAME,
+    password: process.env.PDB_PASSWORD,
+    database: process.env.PDB_NAME,
+    host: process.env.PDB_HOST,
```

Comment:
Nice naming scheme 😄 Just remember that if it's going to be deployed to ElasticBeanstalk it would need `RDS_` as the prefix.